src/ILCompiler.ReadyToRun/src/ILCompiler.ReadyToRun.csproj

```diff
@@ -48,9 +50,39 @@
     <Compile Include="ObjectWriter\SectionBuilder.cs" />
     <Compile Include="ObjectWriter\R2RPEBuilder.cs" />
     <Compile Include="ObjectWriter\RelocationHelper.cs" />
+  </ItemGroup>
+
+  <ItemGroup>
     <Compile Include="..\..\Common\src\System\Collections\Generic\ArrayBuilder.cs">
       <Link>Common\ArrayBuilder.cs</Link>
     </Compile>
+    <Compile Include="..\..\Common\src\Internal\Runtime\EEType.Constants.cs">
+      <Link>Common\EEType.Constants.cs</Link>
+    </Compile>
+    <Compile Include="..\..\Common\src\Internal\Runtime\EETypeBuilderHelpers.cs">
+      <Link>Common\EEType.cs</Link>
+    </Compile>
+    <Compile Include="..\..\ILCompiler.Compiler\src\Compiler\JitHelper.cs">
+      <Link>Compiler\JitHelper.cs</Link>
+    </Compile>
+    <Compile Include="..\..\ILCompiler.Compiler\src\Compiler\MemoryHelper.cs">
```

Comment:
Maybe lets just move this file to the JitInterface directory so that we don't have to source-include out of `ILCompiler.Compiler`?

Reply:
Done